webserver/vhs-stormforger/README.md

```diff
@@ -0,0 +1,112 @@
+# VHS to Stormforger Demonstration
+
+This example demonstrates a workflow optimizing a web application using a load test that is automatically generated from recorded HTTP traffic. This HTTP traffic is captured in the application to be optimized using [VHS](https://github.com/rename-this/vhs), the load test is generated using StormForge Performance Testing, and the application is optimized using StormForge Optimize. This workflow has three main phases:
+
+1. Record HTTP traffic using VHS.
+2. Configure load test generation container and Optimize experiment.
+3. Run Optimize experiment with load test.
+
+## Capture Data with VHS
+
+The rest of this writeup will assume that you have recorded some network traffic with VHS, and that the recorded network traffic is stored in a Google Cloud Storage object.
+This network traffic should be captured in an environment that is similar or identical to the application that will be optimized.
+The [VHS repository](https://github.com/rename-this/vhs) contains a demonstration that shows how to use VHS to capture http traffic in a kubernetes cluster running the voting web app.
+It can be find in the subdirectory `[hack/demo](https://github.com/rename-this/vhs/tree/main/hack/demo)`.
+This demo deploys the voting web app with a VHS container in each of voting-service pod (4 pods), generates some load with simple script using `curl`, and captures 5 minutes of load to a Google Cloud Storage bucket.
+
+
+### Notes:
+
+- There is a `vhsdemo` service account on GCS that can be used for this demo, or a new one can be created.
+- This demo will produce four output files, one from each voting service pod. We don't currently have a good way to combine these files.
+- The `[loadtest.sh](http://loadtest.sh)` script that's in the repository did not work consistently for me. If you encounter the same issue, try increasing the sleep time on line 9 of the script (The "Wait for the container to come up" sleep line).
+
+## Data Post-processing and Load Test Generation
+
+At this point, there should be four objects in the specified GCS bucket. Each object is a gzipped-json file containing the traffic captured by a VHS instance and will be named according to the session-id of the VHS session that captured it. One of these files must be post-processed to create a Stormforger load test. This process has several steps:
```

Comment:
That feels very unspecific. Why are there four files? What are they? How do I choose which one to post-process?